source/api_reference/index.html.md.erb

```diff
@@ -102,7 +102,7 @@ Do not add an allow list of IP addresses to your firewall, because GOV.UK Pay’
 
 GOV.UK Pay authenticates API calls with [OAuth2 HTTP bearer tokens](http://tools.ietf.org/html/rfc6750).
 
-When you make an API call, you need to use an “Authorization” HTTP header to provide your API key, with a “Bearer” prefix. For example:
+When you make an API call, you need to use an "Authorization" HTTP header to provide your API key, with a "Bearer" prefix. For example:
```

Comment:
I think these would probably be better surrounded by backticks to make them appear in a fixed-width this-is-code font.